ams/server/src/main/java/com/netease/arctic/server/optimizing/SchedulingPolicy.java

```diff
@@ -23,13 +23,13 @@ public class SchedulingPolicy {
   private static final String BALANCED = "balanced";
 
   private final Map<ServerTableIdentifier, TableRuntime> tableRuntimeMap = new HashMap<>();
-  private final Comparator<TableRuntime> tableSorter;
+  private Comparator<TableRuntime> tableSorter;
   private final Lock tableLock = new ReentrantLock();
 
   public SchedulingPolicy(ResourceGroup group) {
     String schedulingPolicy = Optional.ofNullable(group.getProperties())
```

Comment:
Recommend making `updateTableSorterIfNeeded` more generic, and we call it directly here.

Reply:
Modified in the new commit.